SparkPost/2web2ui
# FE-1187 - Add the Sending/Bounce Verification Page

FE-1187 - Add the Sending/Bounce Verification Page

### What Changed
 - Added the Sending/Bounce Verification Page
 - Updated the condition for showing the sending and bounce section on details page of completely verified domain
 - Reduced prop drilling happening in DomainDetailsPage
 - Added the pending state to all the verification pages
 - Added cypress tests for Sending/Bounce Verification Page

### How To Test
 - Create Sending Domain from /domains/create
 - Select 'Strict Alignment' from Domain Alignment Modal to get to Sending/Bounce Verification Page.
 - Check if it matches mocks - https://sparkpost.invisionapp.com/console/share/CF19LDYPB2/478945612

### To Do
- [x] Change the base of branch to master before merging
